File: Documentation/3.0/Samples/java/src/test/java/net/ravendb/clientapi/commands/howto/GetDatabaseConfiguration.java
package net.ravendb.clientapi.commands.howto;

import net.ravendb.abstractions.json.linq.RavenJObject;
import net.ravendb.client.IDocumentStore;
import net.ravendb.client.document.DocumentStore;


public class GetDatabaseConfiguration {
	@SuppressWarnings("unused")
	private interface IFoo {
		//region database_configuration_1
		RavenJObject getDatabaseConfiguration();
		//endregion
	}

	@SuppressWarnings("unused")
	public GetDatabaseConfiguration() throws Exception {
		try (IDocumentStore store = new DocumentStore()) {
			//region database_configuration_2
			RavenJObject configuration = store
				.getDatabaseCommands()
					.getAdmin()
					.getDatabaseConfiguration();
			//endregion
		}
	}

}

File: Documentation/3.5/Samples/csharp/Raven.Documentation.Samples/ClientApi/Commands/HowTo/GetDatabaseConfiguration.cs
using Raven.Client.Document;
using Raven.Json.Linq;

namespace Raven.Documentation.Samples.ClientApi.Commands.HowTo
{
	public class GetDatabaseConfiguration
	{
		private interface IFoo
		{
			#region database_configuration_1
			RavenJObject GetDatabaseConfiguration();
			#endregion
		}

		public GetDatabaseConfiguration()
		{
			using (var store = new DocumentStore())
			{
				#region database_configuration_2
				RavenJObject configuration = store
					.DatabaseCommands
					.Admin
					.GetDatabaseConfiguration();
				#endregion
			}
		}
	}
}